demos/wandb/settings

```diff
@@ -0,0 +1,4 @@
+[default]
```

Comment:
@elliottabe what is this file for?

Reply:
(Do we need it?)

Reply:
No that was extra that I didn't add to the .gitignore on accident